app/models/maat_api/prosecution_case.rb

```diff
@@ -0,0 +1,175 @@
+module MaatApi
+  class ProsecutionCase
+    attr_reader :hearing, :prosecution_case, :maat_reference
+    delegate :jurisdiction_type, to: :hearing
+
+    def initialize(hearing_body, case_urn, defendant_data, maat_reference)
+      @shared_time = hearing_body[:sharedTime]
+      @case_urn = case_urn
+      @hearing = HmctsCommonPlatform::Hearing.new(hearing_body[:hearing])
+      @prosecution_case = HmctsCommonPlatform::ProsecutionCase.new(defendant_data)
+      @maat_reference = maat_reference
+    end
+
+    attr_reader :case_urn
+
+    def defendant_asn
+      prosecution_case.defendant_arrest_summons_number
+    end
+
+    def cjs_area_code
+      court_centre.oucode_l2_code
+    end
+
+    def case_creation_date
+      @shared_time.to_date.strftime("%Y-%m-%d")
+    end
+
+    def cjs_location
+      court_centre.short_oucode
+    end
+
+    def doc_language
+      # Should we derive this from somewhere else?
```

Comment:
Indeed, it's worth asking Ahilan if we want to get this with `person_defendant.documentation_language_needs` like we do for Court Applications. Let's make a note of it and do it in a separate PR.